### ISSUE TITLE
App Crashes on MacOS when opening Jitsi Video

### ISSUE THREAD

**Issue by himpierre** [RocketChat/Rocket.Chat.Electron#1965]
## My Setup

- Operating System: MacOS Catalina
- App Version: 3.1.1
- Installation type: dmg

- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
I've configured Rocketchat to use an external Jitsi installation (JWT based) as video solution. When I open the app
and hitting the video button the app crashes immediately after confirming the "are you sure...". dialog. 

Here is the interessting part. When i start the app via terminal:
```
$ pwd
/Applications/Rocket.Chat.app/Contents/MacOS
$ ./Rocket.Chat
[bugsnag] Loaded!
```

the app tells me the terminal needs permission for mic and cam. If I agree, jitsi works just fine with the app. So the problem must be the app not asking for permission somehow.

### Current Behavior
App crashes when starting a video.

### Expected Behavior
Well, app should not crash.

**Comment by AmatorPhasma** [comment]
Same on: Mac OS 11.2.1 
with Version App: 3.0.7, 3.1.0, 3.1.1

**Comment by AmatorPhasma** [comment]
here is a error report or 3.0.7:
[rocket problem.txt](https://github.com/RocketChat/Rocket.Chat.Electron/files/6003176/rocket.problem.txt)

Note: on MacOS 11.1 the 3.0.7 version works fine.

**Comment by ghxm** [comment]
possible duplicate of #1912

**Comment by strike65** [comment]
Is there any progress regarding this issue?

**Comment by vladborovtsov** [comment]
I think it now fixed in 3.5.4 as jitsi session starts in default web browser

**Comment by himpierre** [comment]
That's not a fix it's a workaround. :)

Am 22. September 2021 05:35:46 MESZ schrieb "Vlad E. Borovtsov" ***@***.***>:
>I think it now fixed in 3.5.4 as jitsi session starts in default web browser 
>
